source/styles/main.css

```diff
@@ -1,105 +1,106 @@
 .hidden {
```

Comment:
Just a note to any other reviewers- it looks like the only change here was auto-linting.